src/Tools/CoreFx.Tools/UpdateVSConfigurations.cs

```diff
@@ -276,7 +276,8 @@ private void UpdateSolution(ITaskItem solutionRootItem)
                 foreach (var slnProject in projectFolder.Projects)
                 {
                     string projectName = Path.GetFileNameWithoutExtension(slnProject.ProjectPath);
-                    string relativePathFromCurrentDirectory = slnProject.ProjectPath.Replace(solutionRootPath, "");
+                    // Normalize the directory separators to the windows version given these are projects for VS and only work on windows.
+                    string relativePathFromCurrentDirectory = slnProject.ProjectPath.Replace(solutionRootPath, "").Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar);
```

Comment:
LOL... as you can tell I didn't test it on other platforms :)

I didn't realize we didn't switch the slashes. I will follow-up with this some other point.